packages/eslint-config-wpcalypso/index.js

```diff
@@ -109,7 +107,30 @@ module.exports = {
 		// Assumed by default with Babel
 		strict: [ 2, 'never' ],
 		'template-curly-spacing': [ 2, 'always' ],
-		'valid-jsdoc': [ 2, { requireReturn: false } ],
+
+		// jsdoc plugin
+		'jsdoc/check-access': 2,
+		'jsdoc/check-alignment': 2,
+		'jsdoc/check-param-names': 2,
+		'jsdoc/check-tag-names': 0,
```

Comment:
```suggestion
		'jsdoc/check-tag-names': 'off',
```

? (A few more instances below)

Reply:
I was planning to keep things consistent here (`0`, `1`, `2`) then replace all the ints with their respective strings (`"off"`, `"warn"`, `"error"`) in another PR.